.eslintrc.js

```diff
@@ -27,5 +25,14 @@ module.exports = {
      * https://eslint.org/docs/rules/func-names
      */
     "func-names": "off",
+
+    "max-len": ["error", 256, 2, {

```

Comment:
Why alter these rules?  Move your comments above the line they affect vs. at the end.  Lines this long are not good.

Reply:
Good point, I'll focus on changing the code instead of the rules.